src/devices_models/devices/hydro_generation.jl

```diff
@@ -698,7 +725,7 @@ function add_constraints!(
 ) where {V <: PSY.HydroGen, W <: AbstractHydroFormulation, X <: PM.AbstractPowerModel}
     time_steps = get_time_steps(container)
     resolution = get_resolution(container)
-    # Variable not used. Check if needed Sourabh
+    # TODO: Variable not used. Check if needed Sourabh
```

Comment:
@sourabhdalvi please check why we aren't using this.

Reply:
same here, comparing quantities with same units so no need for conversion.